pkg/api/v1/types.go

```diff
@@ -1806,8 +1811,9 @@ type PodSpec struct {
 	// Value must be a positive integer.
 	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,omitempty" protobuf:"varint,5,opt,name=activeDeadlineSeconds"`
 	// Set DNS policy for containers within the pod.
-	// One of 'ClusterFirst' or 'Default'.
-	// Defaults to "ClusterFirst".
+	// One of 'ClusterFirstAlways', 'ClusterFirst' or 'Default'.
+	// Defaults to "ClusterFirst" unless hostNetwork is true, otherwise Default.
```

Comment:
The point being, the comment implies that the default value changes based on hostNetwork, but it doesn't.  The semantic changes.  Sigh.